blog/2022-03-30-2022.4-new-dev-features.md

```diff
@@ -0,0 +1,37 @@
+---
+author: Paulus Schoutsen
+authorURL: https://twitter.com/balloob
+authorImageURL: /img/profile/paulus.jpg
+authorTwitter: balloob
+title: New features for developers in 2022.4
+---
+
+There are a handful of new features for developers in 2022.4.
+
+- [Integration Type]: Integrations can now mark themselves as helper.
+- [Backup platform]: Integrations can now pause work during a backup by implementing the new backup platform
+- hidden_by in entity registry: Entities can now be hidden via the entity registry. These entities should default to not be included when exposing entities to an external UI like voice assistants.
+- [New and updated selectors]: We added a bunch of new selectors to be used in blueprints, services.yaml, and config flows
+- [Selectors in backend flows]: You can now use the `selector` helper in the
```

Comment:
Is the sentence supposed to continue in the next bullet?

Reply:
```suggestion
- [Selectors in backend flows]: You can now use the `selector` helper in your config flows. It is configured and renders just like selectors in services.yaml/blueprints.
```